lib/ossert/fetch/stackoverflow.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+module Ossert
+  module Fetch
+    # Class implementing `StackOverflow` crawler
+    class StackOverflow
+      attr_reader :client, :project
+
+      extend Forwardable
+      def_delegators :project, :community
+
+      BASE_URL = 'https://api.stackexchange.com/2.2/'
+      SEARCH_URL = 'search/advanced'
+      SEARCH_PARAMS = {
+        site: :stackoverflow,
+        tagged: 'ruby',
+        filter: :total
+      }.freeze
+
+      # Public: Create StackOverflow fetcher
+      #
+      # project - an object representing a project
+      #           it should contain:
+      #             - #name method to return String;
+      #             - #community accessor to return
+      #                 and accept a Hash.
+      #
+      # Returns nothing.
+      def initialize(project)
+        @client = SimpleClient.new(BASE_URL)
+        @project = project
+        @cmn_params = SEARCH_PARAMS.merge(q: "\"#{project.name}\"")
```

Comment:
Is there any troubles to use single quotes for project.name inside double quotes?
I've tried it via StackExchange API check utility and it works...